source/core/parameters.cc

```diff
@@ -1987,7 +1987,7 @@ namespace Parameters
         prm.set("Function expression", "0; 0; 0");
       prm.leave_subsection();
 
-      unsigned int max_ib_particles = 1000;
+      unsigned int max_ib_particles = 10;
```

Comment:
Is the `max_ib_particles = 10` still right?

Reply:
yes this is the value on master now!